Dicom API changes for Image replacement (#121201)
Azure/azure-content: articles/healthcare-apis/dicom/dicom-services-conformance-statement-v2.md

* `application/dicom`

> [!NOTE]

> > The Server **will not** coerce or replace attributes that conflict with existing data for POST requests. All data will be stored as provided. For upsert (PUT) requests, the existing data will be replaced by the new data received. 

Reviewer: ```suggestion
> The server won't coerce or replace attributes that conflict with existing data for POST requests. All data is stored as provided. For upsert (PUT) requests, the existing data is replaced by the new data received. 
```
